Common/Orders/MarketOrder.cs

```diff
@@ -42,6 +42,49 @@ public override decimal Value
         }
 
         /// <summary>
+        /// Create update request for pending orders. Null values will be ignored.
+        /// </summary>
+        public UpdateOrderRequest CreateUpdateRequest(int? quantity = null, string tag = null)
+        {
+            return new UpdateOrderRequest
+            {
+                Id = Guid.NewGuid(),
+                OrderId = Id,
+                Created = DateTime.Now,
+                Quantity = quantity ?? Quantity,
+                Tag = tag ?? Tag
+            };
+        }
+
+        /// <summary>
+        /// Create submit request.
+        /// </summary>
+        public static SubmitOrderRequest CreateSubmitRequest(SecurityType securityType, string symbol, int quantity, DateTime time, decimal price = 0, string tag = null)
+        {
```

Comment:
The MarketOrder type requires the price, the others should have price set to zero. With that being said, @jaredbroad and I spoke about changing how this works in the future. Currently the price is needed for the `Value` property on `Order` during the call to `GetSufficientCapitalForOrder`. The new idea is to have an method such as:
```C#
public abstract decimal GetOrderValue(decimal marketPrice)
```
Which all orders will implement, for now let's just leave it alone though, we'll come back to this in due time.